app/components/TrackCard/index.js

```diff
@@ -0,0 +1,103 @@
+/**
+ *
+ * TrackCard
+ *
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Card, Avatar, Skeleton } from 'antd';
+import { injectIntl } from 'react-intl';
+import { compose } from 'redux';
+import { colors, fonts, styles } from '@themes';
+
+import T from '@components/T';
+import If from '../If/index';
+
+const { Meta } = Card;
+
+const Audio = styled.audio`
+  width: 100%;
+`;
+
+const StyleT = styled(T)`
+  && {
+    color: ${colors.artistNameColor};
+    ${fonts.style.standard()};
+  }
+`;
+
+const AlbumNameT = styled(T)`
+  && {
+    color: ${colors.artistNameColor};
+    ${fonts.style.subheading()};
+  }
+`;
+
+const StyledCaption = styled.figcaption`
+  clear: both;
+  ${styles.margin.top(1)}
+  ${styles.margin.left(0.5)}
+`;
+
+const getSecondValue = (item) => {
+  if (item.wrapperType === 'track') {
```

Comment:
```
 if (item.wrapperType === 'track') {
    return item.trackName;
  } 
  return item.collectionName;
```

Reply:
Aah. the redundant else. I always miss this.